packages/wonder-stuff-ci/src/get-mobile-release-logger.ts

```diff
@@ -0,0 +1,79 @@
+import * as winston from "winston";
+import * as gcpLogger from "@google-cloud/logging-winston";
+const {combine, timestamp, json, errors, printf, colorize} = winston.format;
+
+/**
+ * Winston logger used for generating to GCP for the mobile release process (release-raccoon).
+ * @param {string} projectId The GCP project ID to send logs to.
+ * @param {string} logName The name of the log to send logs to in GCP.
+ * @param {string} logLevel The minimum severity level to log.
+ * @param {boolean} enableConsoleLogs If true, logs will be sent to the console as well as GCP.
+ * @param {[key: string]: string} labels K/V metadata that will be add to the logs
+ * @returns {winston.Logger} logger set up to send logs to GCP.
+ */
+export const getMobileReleaseLogger = (
+    args = {
+        projectId: "mobile-365917",
+        logName: "release-raccoon",
+        logLevel: "info",
+        enableConsoleLogs: false,
+        labels: {},
+    },
+): winston.Logger => {
+    // Set up logging configuration to GCP
+    const loggingGCPWinston = new gcpLogger.LoggingWinston({
```

Comment:
Hmm - so we already have some Winston logging logic here: https://github.com/Khan/wonder-stuff/blob/main/packages/wonder-stuff-server/src/get-logging-transport.ts

I wonder if it makes sense to combine these efforts?

Reply:
This is great! I'll definitely combine the efforts. The only change needed is to add this new GCP logging transport and then I don't have to create the logger and all the other stuff in this file.

Reply:
It might be necessary to export the relevant pieces from Wonder Stuff Server if they aren't already, but yes, we should try to share what we can here. I think it's ok that this new package takes a dependency on WS Server.